### DISCUSSION TITLE
Final Project

### DISCUSSION THREAD

**Discussion by periode** [periode/augmenting-gallery#16] [Announcements]
Create a new React glitch project and post your URL here.

**Comment by Yupu-Chen** [comment]
GalleryEverywhere, created by Samuel (@Opal42) & Yupu

[Zip file of the project and the video documentation](https://drive.google.com/drive/folders/1QC5fJhyWFfR62pfQSZTKZl1nT5U-XRZY?usp=sharing)

GalleryEverywhere is an AR application that allows users to curate their own exhibitions and share them through a built-in community and on social media.  We were inspired by the reading on ignorant museums to replicate the experience of knowledge creation and the Van Abbemuseum's DIY Archive, using AR as a means of creating accessibility and allowing everyone to digitally handle and interact with works they would never be able to in real life. 

It is simple and straightforward, both to ensure accessibility and to avoid as far as possible, any implicit meaning-making on our part, externally imposing certain views or understandings of the art on users. Instead, users are able to choose any piece from our selection, view some basic factual information about it, and if they please, arrange it digitally on the space around them using AR. Once users have arranged all the art they wish, then they are able to create an "exhibition", with their own title, description, and images accompanying it. After creating an exhibition, it is shared with the rest of the world, and others users can view it, while sharing their own. 

One of the key challenges faced (and still yet to be resolved) with this project is how to best "share" the exhibitions users create. Our current route is one of practicality--it is difficult to image a world where we could fully capture then reproduce the spatial data of every user's exhibition. Yet, this implementation is sorely lacking in a true sense of spatiality, which is critical to how these exhibitions are arranged. A potential path forward may be to ask users to build a 360-degree static image of their exhibition, using spliced photographs akin to how some applications stitch together panoramas. The technology for building this app was reasonably straightforward, and if it were to be built out into a full app, managing the server design, and how the community interacts with one another, especially in sharing their experiences, would be the utmost priority.    

This is the [Figma Prototype](https://www.figma.com/file/nUItkQYC7b0m3ze4JFzamg/Augmenting-Gallery?type=design&node-id=0%3A1&t=PAzqXdCzxn3a4LbD-1) and this is the [Canva presenatation](https://www.canva.com/design/DAFhxdlQaiI/lLs2q0q032cw_f6lL1A6pg/edit?utm_content=DAFhxdlQaiI&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton)

**Comment by emersonmack** [comment]
[Here](https://drive.google.com/drive/folders/1hqBiE39Muuwr9kwUbvtgMlYCnIse7HPI?usp=sharing) is a link to all of our content.

An Intro to the History of Music Technology is an educational app that teaches audiences a basic understanding of how music technology has evolved from the 1920s to the 2010s. The app uses a series of screen space menus to reveal information as the user progresses through the app. Each of the ten decades gives the option of seeing the objects in an AR space. This application allows audiences to experience and view these different technologies in a 3D realm. 

This project taught us so many new skills and built off our knowledge from past projects. The biggest struggle we experienced with this project was understanding our limitations. We knew we could not include every decade from the beginning of musical history, nor every object from each chosen decade, nor claim that this was a completed collection. To resolve these issues, we constricted ourselves to one century and introduced the collection as an introduction to the topic. Another issue we had was working between screen-space, AR space, and the integration between our graphic design and Unity’s interface. We successfully combined these interfaces into one concise app using scripts, buttons, and AR. 

In the end, this project allowed us to overcome the steep learning curve of Unity and successfully achieve a prototype of our idea. I am happy with how it finished despite having notes and ideas on how to continue to develop this app.

**Comment by mariamabdelrazekk** [comment]
Mariam & Dania "Hooked"

Link to drive containing project: https://drive.google.com/drive/folders/1_pjaPANn23flmUTN1zWeQ5Mp1QmS40NS?usp=sharing
Link to our video: https://youtu.be/iXPfhAfiIf4

Our application was designed as a learning experience fro those who are interested in crocheting and do not know where to start. we were inspired and heavily influenced by The World Famous Crochet Museum in California. Our initial though was how the museum was not accessible to many people and so our application takes crochet objects and displays them using Augmented Reality to allow people to take a closer look at the crochet objects without having to physically be in the museum space. 

We mainly use buttons to navigate through the application, the first page offers two options, either to get more information about crochet and the difference between it and knitting, and another button that immediately takes the user to the AR space where the user can choose from a cluster of three objects what they want to see and learn about. Then, they are able to see the objects and information about them. 

This project was a huge learning experience for us; there are some things that we tried to implement but could not, for example videos, or links to videos. Moreover, we had an issue with the screen size as my computer did not have an option to work with an iPhone 13 screen size, so all the scenes developed on (mariam's) laptop are smaller in screen size than those of Dania. We would like to add an option for people to add their own objects as well. I think this project challenged our critical thinking skills as much as our skills with unity. Completing the whole idea and piecing it together definitely was not an easy task.

**Comment by hannah-y-zhao** [comment]
[Video demo](https://drive.google.com/file/d/16w57FD5G9RL3Cd64jkfKiKU8HuSA2kqf/view?resourcekey)
[Zipped file](https://drive.google.com/file/d/1SE-GnFsoZ-nwBef4_vVBjdaEoyL1D4-j/view?usp=sharing)
[Presentation](https://docs.google.com/presentation/d/1QJNGt59uL1EuNogOfh6prtLZnSetAbB4q2S_jOyoO9k/edit?usp=sharing)

Komposition by Aakarsh and Hannah
Komposition is an app that visualizes German compound words in a surrealist fashion. It involves creating individual compound words and then viewing each different word as a unique 3D model in AR. Users can discover a whole collection of compound words, and even learn of those that are actually used in the German language. The intention behind this app was to introduce people to linguistic absurdities that they might otherwise have never thought of.

Originally, we were thinking about using AI generated models to allow the users to have more options, and to be more visually unique for each word; however, the technology isn't quite at that level yet. We then thought about creating each model and texture assignment in code, yet we were worried about potential complications and believed that for this project, it would be easier to create many prefabs and just have them named correctly. If we developed this app further, we were interested in the idea of using some sort of geolocation data to gamify it a bit more so users would actually have to "collect" words to combine.

Overall, this project was a great experience and we are really happy with the end result. It's wildly different from our 1st or 2nd or even 5th or 6th ideas (we brought up a lot of potential inspirations and topics) but we created what we set out to create.

**Comment by Caythian** [comment]
https://drive.google.com/drive/folders/16we_QSMNGQy3hSMbWkXZ_cs0rVNmgAsR?usp=sharing

For the final project, I made a website analyzing the game design choices contributing to immersive gaming, using "Legend of Zelda: Tears of the Kingdom" as the case study. I delve deeper into the relationship between immersive games and escapism, about whether gaming is an escapist entertainment that is entirely separated from reality or not. Due to the limited working time, I deducted a whole session about the potentiality of adapting gaming into education, mental health therapy, and general skill and language learning (but you can still see traces of it inside some text and commented lines). 

I created this game driven not only by my interest in gaming but also to present a fresh and positive perspective on video games as a hopeful avenue for societal advancement, viewing them as more than mere entertainment. While the current scope of the pages is limited in fully exploring this concept, this project has truly sparked my interest in furthering my study in this field.

There are two challenges I found during the process: 1)Inserting a game map into a leaflet map is a real pain. I scrapped materials and codes from all the platforms I could search. 2)Adoption to mobile devices: the ImageSection function I used could not be adopted to the mobile no matter how. In the end, I sacrificed the scrolling effect for better viewing on mobile.

**Comment by alkashiwag** [comment]
https://drive.google.com/file/d/15ph4Gnp114OZK9BLgDI073OAeFx4u7Ej/view?usp=drive_link
https://drive.google.com/file/d/1GIOjP1_6wpraUMjDJk-Dp_iXxJ6ogtye/view?usp=drive_link

Our group created a site that explores a variety of trends and controversies in album cover artworks in western music. Across many decades and genres, the albums explored fall into four main categories: sexuality/nudity, violence, politics, and substances. Understanding more about these albums and their artwork helps us to have a peak into what was socially acceptable, what became socially acceptable, and what isn't socially acceptable. Since music is such a large part of popular culture, it can a little light into the depictions of media consumptions.

Based on the four main categories, we selected a variety of albums and displayed them with their artist and genre. Some of the albums have well-known controversies and others do not. Some of the albums have some context provided, but not all. This allows the visitors to form their own thoughts about how the artwork relates to the current times or why it's considered so controversial.

One improvement that we would have liked to implement is a better integration of audio. The ideal version of the website would be able to play audio from a song of the subject album or single when the pop-up is open and you click on an icon, but this was really difficult to implement.

**Comment by Ambraaaliu** [comment]
https://final-project-dear-debussy.glitch.me/
https://drive.google.com/file/d/1svgskbcKss3IquWlng9oRddfjxIoEw24/view?usp=sharing


This site is made by Debussy with three contributors, who decided to promote Debussy in the current time, they decided to collaborate classic with modern. Especially, there are 3 different themes of the album, Children, Water, and Night, chosen by us. There are also several performers from all around the world in the recording and videos. 

Our web page covers a lot of details that we have designed with care, and we hope to really get visitors into the fact that this is a website from the same period as Debussy's music, where we are selling Debussy's new albums and selling his peripherals. (The peripherals are inspired by Kpop). Also our album web design is completely different and we hope it will give the user a more immersive experience.

We have improved a lot of details of the design, improved the uniformity of the web page and refined the details. If there is still time we hope to enrich the page even more.

**Reply by chaerin-d** [reply]
Recorded Screen:
Laptop Ver. [https://drive.google.com/file/d/1HbpL3YAxCM75tBgv2I0xOLG0pGgr7nSg/view?usp=sharing](url)
Phone Ver. [https://drive.google.com/file/d/1LK48152rKGNPLAmzmqv6l8TZvRIgYJhA/view?usp=sharing](url)

**Comment by zbyiii** [comment]
https://drive.google.com/drive/folders/1jrsUBcI_eC7sKnBITj3XGlpT-6cOc3BA?usp=sharing

In this project, I developed a website with a primary focus on astrology. The website encompasses introductory sections on both Western astrology and Chinese Purple Star astrology, a comparative overview, and an exploration into the psychology underpinning beliefs in astrology.

During the development phase, a notable challenge emerged in adapting the media query for varying screen sizes. In the coding process, I aligned the code and the website side by side, resulting in a window size that occupied only half the screen. Designing within this confined space led to unexpected layout discrepancies when viewed in full-screen mode.

Given more time, I aspire to enhance the website by incorporating additional interactive elements. Currently, the life chart and star chart calculations redirect to external links, but with additional time, I aim to develop these features internally. Furthermore, I plan to ensure visual consistency across the website by generating personalized charts and illustrations to maintain a cohesive visual style.

**Comment by liunikolai** [comment]
https://drive.google.com/drive/folders/1DAotH5hYpBONoxoY-sNr0qiMOlsdRGIN?usp=drive_link

We created a website that explores the world of techno in Berlin since the fall of the wall. It is an idiosyncratic world for sure. We split it into three categories, fashion, the sound, and drugs/safety, three massive parts of the scene. The visitor will learn about It through blurbs of text. 

The visitor is automatically directed towards the menu on the top left which lets them see each of the three categories. They will see floating objects which we believe encapsulate the culture right now. 

I think if we had the opportunity to work on this more, we definitely would add more research and make it more mobile compatible. Right now it is super obvious that it was only made for the desktop.

**Comment by el-dah** [comment]
How We Built Our Project (Eldah & Gabriel)
Link to project: https://new-berghain.glitch.me/

Our goal was to design an engaging and interactive experience that captures the stories that developed the widespread intrigue of Berlin’s Berghain club culture. 

Here’s how we went about it:

We used Glitch and combined HTML, CSS, and JavaScript. This was ideal for this project because it enabled us to focus on creativity and functionality without worrying about the complexities of hosting the site, among other things.

The foundation of our project was built using HTML to organizes the content and structure of the site. HTML allowed us to create essential elements like a homepage, an interactive entry simulation, and other sections where users could engage with the project. We set up forms and buttons to mimic the experience of being at Berghain’s door and crossing into different rooms at Berghain, respectively.

To capture the aesthetic of Berghain, we relied on CSS to style the website. The design is inspired by Berghain’s industrial and minimalist vibe, with dark backgrounds, bold typography, and subtle animations. We used CSS to ensure the site also worked well on different device screens and sizes. By adding hover effects and animations, we gave the site a more responsive feel, mimicking the engagement of a social environment.

One of the more interesting parts was creating interactive features using JavaScript. We designed a random outcome generator for the “door policy” simulation, where users could experience the thrill (and frustration) of trying to “get in.” By combining simple programming techniques with creative thinking, we made this interaction feel authentic and unpredictable—like Berghain itself.

Other considerations

To truly emulate the Berghain experience, we hoped to focus on more than just visuals and interactivity. One idea was to add sound to parts of the site but this proved challenging in our timeline as well as potentially distracting/obtrusive to the user. We still wanted the site to feel immersive and reflective of the club’s cultural values. The all black background is meant to create a sense of being in a dark club or room, with the text looking almost suspended or not fixed into a totally clear place. We were somewhat inspired by the “no photography” rule in Berghain which makes capturing the experience more about memory. As for indoor activity, we wanted to keep it simple with relatively linear buttons to transfer pages. We also ensured that the random outcomes in the entry simulation were balanced to maintain a sense of fairness and fun, and perhaps intrigue.

One of the trickiest aspects was fine-tuning the randomness of the “door policy” feature. While we wanted it to feel unpredictable, we also needed to avoid frustration for users who might be repeatedly not into the website — which would be unhelpful to our goal of having users engage with the site’s content. We adjusted the underlying logic to ensure a mix of outcomes that felt natural and engaging, with a much higher ratio of being allowed to enter the site than not.

Another challenge was staying true to the aesthetic and cultural significance of Berghain while adapting it for a simplified digital format. And we had to carefully balance paying homage to the lore and secrecy with stripped down accessibility, ensuring the project was understandable to those curious about Berghain’s enduring legacy as well as resonant with those who have some more open knowledge of Berghain.

This project was a learning experience. Throughout the process, we definitely honed our skills in web development, alongside problem-solving (narrowing down choices as to what is necessary and feasible), and creative collaboration (mixing personal takes with more research-oriented ones). We also deepened our understanding of Berlin’s club culture as a whole and the important role spaces like Berghain play in fostering community and expression across arenas, not just nightlife.

**Reply by periode** [reply]
The starting point of the project of investigating berghain as a cultural icon is a very good one, and your presentation showed that well. However, the website in itself, besides the very nice "gatekeeping" interaction at the beginning, does not entirely do justice to these themes. The research section is good, but could have added a bit more links either to your other pages (e.g. fashion, news), or to the references directly. On the architecture and fashion page, I would have also added more images, references, and either some introductory text on the architecture page (the intro text on the fashion page is also a bit distracting). Overall, this project would really shine with the addition of hyperlinks, stating out the broad themes (e.g. from queer to mainstream?), and a better text layout (less line-height, not centered, and with clearer headings).

- UI Design: B+
- Implementation: A-
- Media & Interactivity: B+
- Research: A-

Final grade: A-

**Comment by marianapazzi** [comment]
https://tijuana-game.glitch.me

Kaveh and Mariana 

**Project Overview:**
For our final project, we created an interactive web-based game that immerses the user in the experience of attempting to cross the US-Mexico border into Tijuana. The game begins with the user filling out a virtual passport, which highlights the various prejudices and challenges faced by those crossing the border, such as discrimination based on name, skin color, gender, time of entry, and religious accessories. This passport experience sets the stage for the user's journey.

**Crossing the Border:**
After completing the passport, the user must engage in a conversation with an intimidating CBP (Customs and Border Protection) officer, simulating the tense and often unfriendly encounters many foreigners face at the border. Regardless of the user's responses, they are ultimately rejected entry into the US 100% of the time, reflecting the harsh realities and uncertainties faced by those seeking a better life across the border.

**Cultural Immersion:**
Upon rejection, the user is redirected to a video of Manu Chao's live performance of "Welcome to Tijuana" at a concert. This cultural artifact encapsulates the stereotypical perception of Tijuana as a city of "tequila, sexo, marihuana." By incorporating this video, we aimed to convey the fever dream-like atmosphere of Tijuana, where seemingly unrelated stories and characters are deeply interconnected.

**Exploring Tijuana's Stories:**
Following the video, the user is presented with a map of Tijuana and can choose to explore the stories of three distinct characters:

- Livio Santini, an Italian immigrant and the creator of the Caesar salad
- Jorge Hank, a millionaire founder of Tijuana who owns the Casino Caliente and has significantly influenced the city's hedonistic culture
- Rosaline, a fictional character based on true stories of female immigrants who come to Tijuana seeking a better future but often end up as sex workers

After delving into each character's narrative, the user can return to the map to explore the other stories.

**Project Rationale:**
Our goal with this project was to provide users with an immersive and thought-provoking experience that captures the essence of Tijuana and the challenges faced by those who find themselves there. By simulating the border crossing process and introducing users to a diverse cast of characters, we aimed to challenge preconceived notions of Tijuana as merely a shadowy, mysterious place. Instead, we sought to portray it as a vibrant, evolving city with a complex history that continues to unfold.

**Reply by periode** [reply]
The approach you proposed of approaching Tijuana through the lens of the stereotype, in order to deconstruct these stereotypes, is very compelling. The dramaturgy of failing to go through the passport control also works very well, and the youtube video is a good view of video (you could have thought of adding more sounds to reflect what one might hear when they're in Tijuana). The dialogues are good way to communicate information, but you could have polished the design and interaction a bit more, as the layout is still slightly messy, and hinders readability. Finally, a mentioned in class, the project would also have benefitted from being a bit more explicit about _the places_ where you encounter the characters: which neighborhoods are they in? What kinds of landmarks? How old are they? Which kinds of people live/hang out there? Combining personas and places could really make the experience of Tijuana shine.

- UI Design: A-
- Implementation: A-
- Media & Interactivity: A
- Research: A

Final grade: A

**Comment by harrydnewman** [comment]
Harry and Cate

https://github.com/harrydnewman/ChatArchive

Our write up is in the read me on GitHub

**Reply by periode** [reply]
The website and presentation were both very compelling and well done! Your chosen topic is very rich (a bit too rich, as our guests mentioned), and you did a good job at designing and implementing a very diegetic interface. In terms of content, I would have added an intro that is a bit more concise (after the iMessage scene) that would have included some in-text links to other pages, to reinforce the site architecture to your vistors. As for design, I agree with the suggestion you mentioned: a first click on each icon on the graphs would have helped the visitor navigate better without having to go back and forth at all times (other pages do include links, and that's great!). Finally, the "Future of Chat" page could have been a bit clearer with a styling on each headings (Metaverse, AR, Personalization).

- UI Design: A
- Implementation: A
- Media & Interaction: A-
- Research: A

Final grade: A

**Comment by SophiaC-R** [comment]
Jacie and Sophia 

https://cultured-juniper-silverfish.glitch.me/
[Augmenting FInal Write Up.pdf](https://github.com/user-attachments/files/18094272/Augmenting.FInal.Write.Up.pdf)

**Reply by periode** [reply]
The transformation of architecture is a good point to start from, but is indeed very broad. When visitors arrive on your website, they will not have had the chance to hear your presentation, and you would need to have more pointers on the website to make it accessible and understandable. For instance, the title "Transformation of German Spaces" is very broad, and can mean a lot of different things. Since you address a specific kind of transformation (building re-purposing), it would have been better to focus on just this part. Additionally, your website is also missing some hyperlinks (which makes the heart of a website!). Instead of putting screenshots of the sections of other websites, you could directly link to those.

- UI Design: B+
- Implementation: A
- Media & Interactivity: B+
- Research: B+

Final grade: B+

**Comment by lucyking140** [comment]
Here is my final project link: https://our-machines.netlify.app/

**Comment by mayamcduffie** [comment]
here's my final project link: https://maya-final-biking.glitch.me/

**Comment by readesmith** [comment]
here's my final project link: https://zine-project.glitch.me/

**Comment by nixophobia** [comment]
this is my final project link: https://game-over-time.glitch.me
